### PR TITLE
Added Pantip, Blockdit | Added Joox | Bugs Fixed, Update Snowboltz

### PR DESCRIPTION
**Pull request by Yusaki-Akito**

### CODE REVIEW

**Comment by Yusaki-Akito** [comment]
Preview
- [Home page] 
image

File: websites/P/Pantip/presence.ts [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    presenceData.details = "หน้าที่ไม่ทราบ";
+  }
+}
+
+if (presenceData.details == null) {
+  presence.setTrayTitle();
+  presence.setActivity();
+} else {
+  presence.setActivity(presenceData);
+  //console.log(presenceData);
```
Comment (Bas950) [inline websites/P/Pantip/presence.ts:175]: remove that console log comment

**Comment by Yusaki-Akito** [comment]
I have update some useless thing and some bugs fixed on snowboltz sorry but thank you for  participantion!

**Comment by Yusaki-Akito** [comment]
**Bugs Fixed, Update Useless | SnowBoltz**
- [Fixed not show presences when not login]
image

- [Fixed Show wrong page | Home into coupon]
image

- [Added Some presences on admin page | State show the member ID for they can keep tracking what we do for support]
image

> **Add | Blockdit**
- [Home page]
image

- [Bookmark]
image

- [Explore]
image

- [Articles]
image

- [Searching]
image

**Comment by Yusaki-Akito** [comment]
> Change on Blockdit
[+] Change Reading | Articles to Reading on articles

> Add Joox

- [Listening]
image

- [Paused]
image

File: websites/B/presence.ts [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+if (presenceData.details == null) {
+  presence.setTrayTitle();
+  presence.setActivity();
+} else {
+  presence.setActivity(presenceData);
+  //console.log(presenceData);
+}
+});
+
+
```
Comment (Bas950) [inline websites/B/presence.ts:257]: Pretty sure this file shouldnt be here xD

Comment (Bas950) [reply websites/B/presence.ts:257]: websites/B/presence.ts

**Comment by doomlerd** [comment]
also make sure that "B" image has a transparent background or something, just plain white doesn't look good imo

File: websites/B/presence.ts (hunk 2 above)
Comment (Yusaki-Akito) [reply websites/B/presence.ts:257]: OOF Sorry

**Pull request merged**